### PR TITLE
Add spinwheels 

### PR DESCRIPTION
**Pull request by Ali-Anomma**
#### Summary
Content "Adds spinwheels to decrase time needed for crafting rags,ropes and strings."

#### Purpose of change
Hand crafting ropes is so tiring and a survivor can build a spinwheel to make it easier and quick

#### Describe the solution
Add spinwheel,A machine can be build with decent skills and acts as a fake tool.i added some recipes which uses spinwheel, its time significantly lowered and have batch craft option.

#### Describe alternatives you've considered
Maybe making spinwheel interactable,and making strings on spinwheels menu.

#### Testing
i changed recipes,furnitures and constructions locally,worked fine,uploaded then.

#### Additional context
Picture of spinwheel:
external-content duckduckgo com

### CODE REVIEW

**Comment by I-am-Erk** [comment]
we should also add some of the pre-industrial tools for ropemaking. There are simpler ones: 
image

That can be another PR.

**Comment by Ali-Anomma** [comment]
> 
> 
> we should also add some of the pre-industrial tools for ropemaking. There are simpler ones:
> image
> 
> That can be another PR.
> 
> You are missing a space in your PR description line that is causing it to fail the test.

how to rerun tests to pass the pr validator?
what is name of tool? i can make some research of it.

**Comment by Maleclypse** [comment]
> > we should also add some of the pre-industrial tools for ropemaking. There are simpler ones:
> > image
> > That can be another PR.
> > You are missing a space in your PR description line that is causing it to fail the test.
> 
> how to rerun tests to pass the pr validator?

You edit the top box.  It looked like a missing space so I've edited the missing space in.    To fix json validation follow the instructions at https://dev.narc.ro/cataclysm/format.html.

File: data/json/recipes/other/cords_and_ropes.json [hunk 1]
```diff
@@ -81,6 +162,22 @@
...
+  {
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "result": "rope_30",
+	"id_suffix":"spinwheel_ropes",
+	"batch_time_factors": [ 70, 1 ],
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_MATERIALS",
+    "skill_used": "tailor",
+    "time": "1 m",
```
Comment (I-am-Erk) [inline data/json/recipes/other/cords_and_ropes.json:174]: I think these times are a bit short. Getting the fibers onto the wheel alone would probably take a minute or two.

**Comment by Maleclypse** [comment]
>  To fix json validation follow the instructions at https://dev.narc.ro/cataclysm/format.html.

You asked on discord what to do for json validation.   Go to the page above, copy the entire file into the webpage, hit the lint button, if no popups occur, copy the entire file back out of the webpage and overwrite your file on your computer with it, save that.   Do this for every file then upload the commit with the new files to this PR.

**Comment by TealcOneill** [comment]
@Ali-Anomma Are you able/planning to finish this?

**Comment by I-am-Erk** [comment]
I am closing this but this is still something i'd like to see.
A new implementation should remove the "construction" option and instead have the spinwheel be deployable, as well as fix the issues commented on here.

Since that's very little, I am going to put this in the "abandoned but desired" pile for someone, perhaps me, to pick up later.

**Pull request closed without merging**